From: crvi <crvisqr@gmail.com>
Date: Mon, 3 Feb 2020 21:12:41 +0530
Subject: plugins: Update website links

Replace www.rhythmbox.org with updated wiki.gnome.org links

Bug: https://gitlab.gnome.org/GNOME/rhythmbox/-/issues/1751
Bug-Debian: https://bugs.debian.org/964946
Applied-upstream: 3.4.5, commit:64c07859c936df1bc739f21b87f7a56e6f8dd161
---
 plugins/android/android.plugin.in                     | 2 +-
 plugins/artsearch/artsearch.plugin.in                 | 2 +-
 plugins/audiocd/audiocd.plugin.in                     | 2 +-
 plugins/audioscrobbler/audioscrobbler.plugin.in       | 2 +-
 plugins/brasero-disc-recorder/cd-recorder.plugin.in   | 2 +-
 plugins/context/context.plugin.in                     | 2 +-
 plugins/daap/daap.plugin.in                           | 2 +-
 plugins/dbus-media-server/dbus-media-server.plugin.in | 2 +-
 plugins/fmradio/fmradio.plugin.in                     | 2 +-
 plugins/generic-player/generic-player.plugin.in       | 2 +-
 plugins/grilo/grilo.plugin.in                         | 2 +-
 plugins/ipod/ipod.plugin.in                           | 2 +-
 plugins/iradio/iradio.plugin.in                       | 2 +-
 plugins/lirc/rblirc.plugin.in                         | 2 +-
 plugins/lyrics/lyrics.plugin.in                       | 2 +-
 plugins/magnatune/magnatune.plugin.in                 | 2 +-
 plugins/mmkeys/mmkeys.plugin.in                       | 2 +-
 plugins/mpris/mpris.plugin.in                         | 2 +-
 plugins/mtpdevice/mtpdevice.plugin.in                 | 2 +-
 plugins/notification/notification.plugin.in           | 2 +-
 plugins/power-manager/power-manager.plugin.in         | 2 +-
 plugins/pythonconsole/pythonconsole.plugin.in         | 2 +-
 plugins/rbzeitgeist/rbzeitgeist.plugin.in             | 2 +-
 plugins/replaygain/replaygain.plugin.in               | 2 +-
 plugins/soundcloud/soundcloud.plugin.in               | 2 +-
 plugins/webremote/webremote.plugin.in                 | 2 +-
 26 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/plugins/android/android.plugin.in b/plugins/android/android.plugin.in
index 5f7ed62..353c4e2 100644
--- a/plugins/android/android.plugin.in
+++ b/plugins/android/android.plugin.in
@@ -6,7 +6,7 @@ _Name=Android devices
 _Description=Support for Android 4.0+ devices (via MTP)
 Authors=Jonathan Matthew
 Copyright=Copyright © 2015 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/artsearch/artsearch.plugin.in b/plugins/artsearch/artsearch.plugin.in
index 5ecac89..5f4929d 100644
--- a/plugins/artsearch/artsearch.plugin.in
+++ b/plugins/artsearch/artsearch.plugin.in
@@ -7,7 +7,7 @@ _Name=Cover art search
 _Description=Fetch album covers from the Internet
 Authors=Jonathan Matthew <jonathan@d14n.org>
 Copyright=Copyright © 2011 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/audiocd/audiocd.plugin.in b/plugins/audiocd/audiocd.plugin.in
index ad334ba..6db8239 100644
--- a/plugins/audiocd/audiocd.plugin.in
+++ b/plugins/audiocd/audiocd.plugin.in
@@ -6,4 +6,4 @@ _Name=Audio CD Player
 _Description=Support for playing of audio CDs as music source
 Authors=James Livingston, Jonathan Matthew
 Copyright=Copyright © 2005-2006 James Livingston, Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/audioscrobbler/audioscrobbler.plugin.in b/plugins/audioscrobbler/audioscrobbler.plugin.in
index 5772a3e..c802c7c 100644
--- a/plugins/audioscrobbler/audioscrobbler.plugin.in
+++ b/plugins/audioscrobbler/audioscrobbler.plugin.in
@@ -5,7 +5,7 @@ _Name=Last.fm
 _Description=Submits song information to Last.fm and plays Last.fm radio streams
 Authors=Alex Revo <xiphoidappendix@gmail.com>;Ruben Vermeersch <ruben@Lambda1.be>
 Copyright=Copyright © 2005-2006 Alex Revo, Ruben Vermeersch
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/brasero-disc-recorder/cd-recorder.plugin.in b/plugins/brasero-disc-recorder/cd-recorder.plugin.in
index 7793bdc..e8b06f5 100644
--- a/plugins/brasero-disc-recorder/cd-recorder.plugin.in
+++ b/plugins/brasero-disc-recorder/cd-recorder.plugin.in
@@ -5,7 +5,7 @@ _Name=Audio CD Recorder
 _Description=Record audio CDs from playlists and duplicate audio CDs
 Authors=William Jon McCann, Rouquier Philippe
 Copyright=Copyright © 2006 William Jon McCann, © 2008-2009 Rouquier Philippe
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/context/context.plugin.in b/plugins/context/context.plugin.in
index 0098be4..86271e0 100644
--- a/plugins/context/context.plugin.in
+++ b/plugins/context/context.plugin.in
@@ -7,4 +7,4 @@ _Name=Context Pane
 _Description=Show information related to the currently playing artist and song.
 Authors=John Iacona <plate0salad@gmail.com>
 Copyright=Copyright © 2009 John Iacona
-Website=http://rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/daap/daap.plugin.in b/plugins/daap/daap.plugin.in
index 93e16fd..fbd5384 100644
--- a/plugins/daap/daap.plugin.in
+++ b/plugins/daap/daap.plugin.in
@@ -5,7 +5,7 @@ _Name=DAAP Music Sharing
 _Description=Share music and play shared music on your local network
 Authors=Charles Schmidt  <cschmidt2@emich.edu>, the Rhythmbox Developers
 Copyright=Copyright © 2005-2006 Charles Schmidt, the Rhythmbox Developers
-Website=http://www.rhythmbox.org
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/dbus-media-server/dbus-media-server.plugin.in b/plugins/dbus-media-server/dbus-media-server.plugin.in
index 14f96d8..8930708 100644
--- a/plugins/dbus-media-server/dbus-media-server.plugin.in
+++ b/plugins/dbus-media-server/dbus-media-server.plugin.in
@@ -5,7 +5,7 @@ _Name=MediaServer2 D-Bus interface
 _Description=Provides an implementation of the MediaServer2 D-Bus interface specification
 Authors=Jonathan Matthew
 Copyright=Copyright © 2010 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/fmradio/fmradio.plugin.in b/plugins/fmradio/fmradio.plugin.in
index b0a70ab..2c1d9ba 100644
--- a/plugins/fmradio/fmradio.plugin.in
+++ b/plugins/fmradio/fmradio.plugin.in
@@ -5,4 +5,4 @@ _Name=FM Radio
 _Description=Support for FM radio broadcasting services
 Authors=James Henstridge
 Copyright=Copyright © 2007 James Henstridge
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/generic-player/generic-player.plugin.in b/plugins/generic-player/generic-player.plugin.in
index 2523045..927b45e 100644
--- a/plugins/generic-player/generic-player.plugin.in
+++ b/plugins/generic-player/generic-player.plugin.in
@@ -6,4 +6,4 @@ _Name=Portable Players
 _Description=Support for generic audio player devices (plus PSP and Nokia 770)
 Authors=James Livingston, Jonathan Matthew
 Copyright=Copyright © 2005-2006 James Livingston, Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/grilo/grilo.plugin.in b/plugins/grilo/grilo.plugin.in
index 38df8dd..20b8af0 100644
--- a/plugins/grilo/grilo.plugin.in
+++ b/plugins/grilo/grilo.plugin.in
@@ -5,4 +5,4 @@ _Name=Grilo media browser
 _Description=Browse various local and Internet media sources
 Authors=Jonathan Matthew
 Copyright=Copyright © 2011 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/ipod/ipod.plugin.in b/plugins/ipod/ipod.plugin.in
index a9c8902..b6a27ca 100644
--- a/plugins/ipod/ipod.plugin.in
+++ b/plugins/ipod/ipod.plugin.in
@@ -5,7 +5,7 @@ _Name=Portable Players - iPod
 _Description=Support for Apple iPod devices (show the content, play from device)
 Authors=James Livingston, Christophe Fergeau
 Copyright=Copyright © 2006 James Livingston
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/iradio/iradio.plugin.in b/plugins/iradio/iradio.plugin.in
index 19013c3..48461dc 100644
--- a/plugins/iradio/iradio.plugin.in
+++ b/plugins/iradio/iradio.plugin.in
@@ -6,4 +6,4 @@ _Name=Internet Radio
 _Description=Support for broadcasting services transmitted via the Internet
 Authors=Rhythmbox authors
 Copyright=Copyright © 2002-2006 Rhythmbox authors
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/lirc/rblirc.plugin.in b/plugins/lirc/rblirc.plugin.in
index 7a96f42..4b4c866 100644
--- a/plugins/lirc/rblirc.plugin.in
+++ b/plugins/lirc/rblirc.plugin.in
@@ -5,4 +5,4 @@ _Name=LIRC
 _Description=Control Rhythmbox using an infrared remote control
 Authors=Jonathan Matthew <jonathan@d14n.org>
 Copyright=Copyright © 2006 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/lyrics/lyrics.plugin.in b/plugins/lyrics/lyrics.plugin.in
index 454d0b1..8b85ed9 100644
--- a/plugins/lyrics/lyrics.plugin.in
+++ b/plugins/lyrics/lyrics.plugin.in
@@ -7,4 +7,4 @@ _Name=Song Lyrics
 _Description=Fetch song lyrics from the Internet
 Authors=Jonathan Matthew, Eduardo Gonzalez, Sirio Bolaños
 Copyright=Copyright © 2005-2007 Jonathan Matthew, Eduardo Gonzalez, Sirio Bolaños
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/magnatune/magnatune.plugin.in b/plugins/magnatune/magnatune.plugin.in
index abcebeb..1d46f3b 100644
--- a/plugins/magnatune/magnatune.plugin.in
+++ b/plugins/magnatune/magnatune.plugin.in
@@ -7,4 +7,4 @@ _Name=Magnatune Store
 _Description=Adds support to Rhythmbox for playing and purchasing from the Magnatune online music store
 Authors=Adam Zimmerman, James Livingston
 Copyright=Copyright © 2006 Adam Zimmerman
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/mmkeys/mmkeys.plugin.in b/plugins/mmkeys/mmkeys.plugin.in
index 1376bcf..597254e 100644
--- a/plugins/mmkeys/mmkeys.plugin.in
+++ b/plugins/mmkeys/mmkeys.plugin.in
@@ -6,4 +6,4 @@ _Name=Media Player Keys
 _Description=Control Rhythmbox using key shortcuts
 Authors=Rhythmbox authors
 Copyright=Copyright © 2002-2007 Rhythmbox authors
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/mpris/mpris.plugin.in b/plugins/mpris/mpris.plugin.in
index a88b230..de23d73 100644
--- a/plugins/mpris/mpris.plugin.in
+++ b/plugins/mpris/mpris.plugin.in
@@ -5,7 +5,7 @@ _Name=MPRIS D-Bus interface
 _Description=Provides an implementation of the MPRIS D-Bus interface specification
 Authors=Jonathan Matthew
 Copyright=Copyright © 2010 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/mtpdevice/mtpdevice.plugin.in b/plugins/mtpdevice/mtpdevice.plugin.in
index c80edec..3ed54ae 100644
--- a/plugins/mtpdevice/mtpdevice.plugin.in
+++ b/plugins/mtpdevice/mtpdevice.plugin.in
@@ -5,7 +5,7 @@ _Name=Portable Players - MTP
 _Description=Support for MTP devices (show the content, transfer, play from device)
 Authors=Peter Grundström
 Copyright=Copyright © 2006-2007 Peter Grundström
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/notification/notification.plugin.in b/plugins/notification/notification.plugin.in
index 9aee0c0..7dfa190 100644
--- a/plugins/notification/notification.plugin.in
+++ b/plugins/notification/notification.plugin.in
@@ -5,7 +5,7 @@ _Name=Notification
 _Description=Notification popups
 Authors=Jonathan Matthew
 Copyright=Copyright © 2010 Jonathan Matthew
-Website=http://www.rhythmbox.org
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 
 [RB]
 InitiallyEnabled=true
diff --git a/plugins/power-manager/power-manager.plugin.in b/plugins/power-manager/power-manager.plugin.in
index eabfac7..3a1da0c 100644
--- a/plugins/power-manager/power-manager.plugin.in
+++ b/plugins/power-manager/power-manager.plugin.in
@@ -6,4 +6,4 @@ _Name=Power Manager
 _Description=Inhibit Power Manager from suspending the machine while playing
 Authors=Jonathan Matthew <jonathan@d14n.org>
 Copyright=Copyright © 2006 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/pythonconsole/pythonconsole.plugin.in b/plugins/pythonconsole/pythonconsole.plugin.in
index de9f4b8..35728bf 100644
--- a/plugins/pythonconsole/pythonconsole.plugin.in
+++ b/plugins/pythonconsole/pythonconsole.plugin.in
@@ -6,4 +6,4 @@ _Name=Python Console
 _Description=Interactive python console
 Authors=Steve Frécinaux <steve@istique.net>
 Copyright=Copyright © 2006 Steve Frécinaux
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/rbzeitgeist/rbzeitgeist.plugin.in b/plugins/rbzeitgeist/rbzeitgeist.plugin.in
index 3d362bb..c4758da 100644
--- a/plugins/rbzeitgeist/rbzeitgeist.plugin.in
+++ b/plugins/rbzeitgeist/rbzeitgeist.plugin.in
@@ -7,4 +7,4 @@ _Name=Zeitgeist
 _Description=Inform Zeitgeist about your activity
 Authors=Markus Korn <thekorn@gmx.de>
 Copyright=Copyright © 2009 Markus Korn
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/replaygain/replaygain.plugin.in b/plugins/replaygain/replaygain.plugin.in
index b717869..a197caa 100644
--- a/plugins/replaygain/replaygain.plugin.in
+++ b/plugins/replaygain/replaygain.plugin.in
@@ -7,4 +7,4 @@ _Name=ReplayGain
 _Description=Use ReplayGain to provide a consistent playback volume
 Authors=Jonathan Matthew
 Copyright=Copyright © 2010 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
diff --git a/plugins/soundcloud/soundcloud.plugin.in b/plugins/soundcloud/soundcloud.plugin.in
index d565817..2e6c2a8 100644
--- a/plugins/soundcloud/soundcloud.plugin.in
+++ b/plugins/soundcloud/soundcloud.plugin.in
@@ -6,5 +6,5 @@ _Name=SoundCloud
 _Description=Browse and play sounds from SoundCloud®
 Authors=Jonathan Matthew
 Copyright=Copyright © 2014 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 Depends=rb
diff --git a/plugins/webremote/webremote.plugin.in b/plugins/webremote/webremote.plugin.in
index 9a3d967..fed1d57 100644
--- a/plugins/webremote/webremote.plugin.in
+++ b/plugins/webremote/webremote.plugin.in
@@ -6,5 +6,5 @@ _Name=Web remote control
 _Description=Control Rhythmbox from a web browser
 Authors=Jonathan Matthew <jonathan@d14n.org>
 Copyright=Copyright © 2016 Jonathan Matthew
-Website=http://www.rhythmbox.org/
+Website=https://wiki.gnome.org/Apps/Rhythmbox
 Depends=rb
